app/models/individual_market/attestation.rb

```diff
@@ -19,13 +19,53 @@ class Attestation
     #   @return [IndividualMarket::Application] The application this attestation belongs to
     embedded_in :application, class_name: 'IndividualMarket::Application'
 
-    # Examples of terms that a primary person/applicant must attest to. Change these as needed.
-    field :enrollment_terms, type: Boolean
-    # field :medicaid_terms, type: Boolean
-    # field :medicaid_insurance_collection_terms, type: Boolean
-    # field :report_change_terms, type: Boolean
-    # field :parent_living_out_of_home_terms, type: Boolean
-    # field :attestation_terms, type: Boolean
-    # field :submission_terms, type: Boolean
+    # @!attribute SIGNER_ROLE_KINDS
+    # @return [Array<String>] Collection of all possible signer roles
+    SIGNER_ROLE_KINDS = %w[admin assister broker consumer].freeze
```

Comment:
what are we going to do for attestations for system submitted applications? things like redeterminations and end of rop - would we copy last attestation or do we need a system attestation OR bypass attestation validation if system submitted?